### PR TITLE
Chapter 4

### PR DESCRIPTION
**Pull request by alexdeas**
ajd chapter 4

### CODE REVIEW

File: chapter04/ajd/exercise3.pl [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    shirt    => 'Olive?',
+    position => 'Islander?',
+    location => 'The Island Country Club',
+  },
+  {
+    name     => 'Mrs. Howell',
+    hat      => 'Chartreuse?',
+    shirt    => 'Apricot?',
+    position => 'Islander?',
+    location => 'The Island Country Club',
```
Comment (dickyince) [inline chapter04/ajd/exercise3.pl:34]: Didn't read the exercise, everyone was meant to start on "The Island"

Comment (alexdeas) [inline chapter04/ajd/exercise3.pl:34]: They did, until I updated the hash like it said.

Comment (ardavey) [inline chapter04/ajd/exercise3.pl:34]: You were also supposed to add the location separately.  Cheater.

File: chapter04/ajd/exercise3.pl [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  },
+);
+
+foreach my $crewmember (@crew) {
+  printf ( "%s at %s\n", @$crewmember{qw( name location )} );
```
Comment (ardavey) [inline chapter04/ajd/exercise3.pl:39]: Slicey slicey

**Pull request merged**